spec/paths/invoices@{id}@void.yaml

```
@@ -4,6 +4,7 @@ post:
   tags:
     - Invoices
   summary: Void an invoice
+  operationId: PostInvoiceVoiding
```

Please note that `void` is a noun.
```suggestion
  operationId: PostInvoiceVoid
```